src/webapi/src/telemetry.rs

```diff
@@ -0,0 +1,39 @@
+// based on https://github.com/LukeMathWalker/zero-to-production/blob/main/src/telemetry.rs
+
+use tracing::subscriber::set_global_default;
+use tracing::Subscriber;
+use tracing_bunyan_formatter::{BunyanFormattingLayer, JsonStorageLayer};
+use tracing_log::LogTracer;
+use tracing_subscriber::fmt::MakeWriter;
+use tracing_subscriber::{layer::SubscriberExt, EnvFilter, Registry};
+
+/// Compose multiple layers into a `tracing`'s subscriber.
+///
+/// # Implementation Notes
+///
+/// We are using `impl Subscriber` as return type to avoid having to spell out the actual
+/// type of the returned subscriber, which is indeed quite complex.
+pub fn get_subscriber<Sink>(
+    name: String,
```

Comment:
If this is always going to be a hardcoded string, you might as well accept a `&str` here and call `.into()` when needed inside this function.

Reply:
Done